test/RandomTest.cpp

```diff
@@ -0,0 +1,47 @@
+//
```

Comment:
If it is not working yet, you don't have to commit it

Reply:
There is error in CMakeList because of which it is not working

Reply:
Please more specific on what errors you are encountering

Reply:
I guess those are mainly because of improper setup of CMakeLists. I'll improve the CMakeLists once and try all tests. If the error still exists, I'll reach out back to you.

Reply:
@merlinvn I have tested all other tests. All are working except `configTest`, `randomTest`

Reply:
Why it is not working? It does not compile or the tests failed or anything else? You should be more precise describe here.

Reply:
It doesn't compile. I am attaching the image of the bug.

![bug_1](https://user-images.githubusercontent.com/20890066/110847927-7e827800-827b-11eb-8dfc-b264ee95de75.JPG)

Reply:
Try to see if you did include the "easyloggingpp" to your test? Refer to my "test" settings,  my test setting works and it should be with yours

Reply:
I included `easyloggingpp`. I think I have included everything as mentioned in your test CMakeList.
I am attaching my CmakeList. Please have a look once if you find anything wrong.

```
find_package(fmt CONFIG REQUIRED)
find_package(GTest CONFIG REQUIRED)
find_package(GSL REQUIRED)
#project(Google_tests)

include_directories(${POMS_SOURCE_DIR}/src ${EXT_PROJECTS_DIR}/easyloggingpp.9.96.4)


set(SRC_TEST test1.cpp CoordinateTest.cpp LocationTest.cpp core/StringHelpersTest.cpp
        core/TimeHelpersTest.cpp RandomTest.cpp ConfigTest.cpp)

add_executable(main_test ${SRC_TEST})
add_dependencies(main_test date args MaSimCore)

#include_directories (${POMS_SOURCE_DIR}/src)
#include_directories(${EXT_PROJECTS_DIR}/easyloggingpp.9.96.4)

target_link_libraries(main_test PRIVATE MaSimCore yaml-cpp fmt::fmt-header-only
        GSL::gsl GSL::gslcblas GTest::gtest GTest::gtest_main)

target_compile_features(main_test PRIVATE cxx_range_for)


add_custom_command(TARGET main_test POST_BUILD
        COMMAND ${CMAKE_COMMAND} -E copy_if_different
        ${POMS_SOURCE_DIR}/test/test_data/input.yml $<TARGET_FILE_DIR:main_test>)
```

Reply:
Alright, try to google "el::base::elStorage" (this is from your error screenshot) and see if you can come up with a solution for it

Reply:
Is it related to `INITIALIZE_EASYLOGGINGPP`?

Reply:
I got it. You have defined it in `main.cpp`, but looks like main() is configured for `Catch`. I have to make changes according to GTest.